selendroid-server/src/main/java/io/selendroid/server/ServerInstrumentation.java

```diff
@@ -56,6 +56,7 @@
   private AndroidWait androidWait = new AndroidWait();
   private PowerManager.WakeLock wakeLock;
   private int serverPort = 8080;
+  private static String automationName = "selendroid";
```

Comment:
does this need to be static? doesn't appear to.

Also, we should find an appropriate place to put the "selendroid" and "uiautomator" strings and then reference them throughout this change.